neuralprophet/time_dataset.py

```diff
@@ -512,43 +512,37 @@ def make_events_features(df, config_events: Optional[configure.ConfigEvents] = N
     additive_events = pd.DataFrame()
     multiplicative_events = pd.DataFrame()
 
+    # helper function to avoid redundant code
+    def create_event_features(event, config):
```

Comment:
The function name might be a little confusing since it is almost the same as `make_events_features()`